kapp_platforms/src/windows/event_loop_windows.rs

```diff
@@ -36,6 +36,41 @@ pub unsafe extern "system" fn window_callback(
         }
         WM_KEYDOWN | WM_SYSKEYDOWN => produce_event(process_key_down(w_param, l_param)),
         WM_KEYUP | WM_SYSKEYUP => produce_event(process_key_up(w_param, l_param)),
+        WM_CHAR => {
+            let character = String::from_utf16(&[w_param as u16])
+                .unwrap()
+                .chars()
+                .next()
+                .unwrap();
+            produce_event(Event::CharacterReceived { character });
+        }
+        WM_IME_STARTCOMPOSITION => {
+            return TRUE as isize;
```

Comment:
Are these `return TRUE as isize;` lines necessary for the composition events? 

I see no mention in the win32 docs about an expected return value.

Reply:
Actually, you're right. `return 0` might suffice there. However, do note that there are also no mentions of an expected return value for WM_IME_COMPOSITION, yet example code I found online seems to distinguish return values of TRUE vs. 0.

Reply:
I was actually thinking maybe no return at all is needed here. Is that not the case?